python/py-CacheControl/Portfile

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-CacheControl
```

Comment:
most, but not all, of the port names are in lower case. From last years GSoC where we started working on a [MacPorts backend](https://github.com/macports/upt-macports/) for [upt](https://framagit.org/upt/upt) it became clear that a standardized naming conventions is important. Admittedly, we didn't reach a broad consensus about it, but using lowercase as this is customary seems reasonable - so that's what I would prefer here. 

If you change that (both the directory name and ```name``` field), you can just add ```python.rootname CacheControl``` under the ```name``` field and the python PG will keep doing TheRightThing.

Reply:
I've changed it over to lowercase as that seems like a good choice.